lib/clamp-link.in

```diff
@@ -157,7 +177,7 @@ do
     fi
 
     OBJS_TO_PROCESS+=( "$ARG" )
-  elif [[ "$ARG" =~ ^-l[^[:space:]]+$ ]] || [[ "$ARG" =~ [^[:space:]]+.a$ ]]; then
+  elif [[ "$ARG" =~ ^-l[^[:space:]]+$ ]] || [[ "$ARG" =~ [^[:space:]]+.a$ ]] || [[ "$ARG" =~ [^[:space:]]+.lo$ ]]; then
```

Comment:
libtool treats .lo as a PIC object; however it looks like bazel treats it as an "always link" library:
https://bazel.build/versions/master/docs/be/c-cpp.html
It's unclear from the dock what an always link library really is so it's quite confusing

Should there be some logic to make the distinction between these 2 usages?